lib/katello/params_parser_wrapper.rb

```diff
@@ -0,0 +1,16 @@
+module Katello
+  class ParamsParserWrapper < ActionDispatch::ParamsParser
```

Comment:
Answered in the lower part of [this](https://github.com/Katello/katello/pull/6640#issuecomment-284220686) comment.